src/util.cr

```diff
@@ -33,5 +33,13 @@ module Wsman
         log.info("Not removing #{path}, does not exist.")
       end
     end
+
+    def self.set_owner(user, group, file, recursive = false)
+      Wsman::Util.cmd("chown", [recursive ? "-R" : "", "#{user}:#{group}", file])
```

Comment:
Crystal stdlib provides an API for this. Please take a look [here](https://crystal-lang.org/api/0.28.0/File.html#chown%28path%3APath%7CString%2Cuid%3AInt%3D-1%2Cgid%3AInt%3D-1%2Cfollow_symlinks%3Dfalse%29-class-method).

The only downside is that it only accepts numberic UID/GID, so we need to figure that out before calling it. Since it increases complexity, we might as well leave it as is, unless it's relatively simple to get UID/GID of a user with native Crystal stdlib calls.